### PR TITLE
add parameter section to README.md

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +71,46 @@ Fedora 25. Other nodesets are currently not present. The module requires at
...
+[limitations](#limitations).
+
+### `$repourl`
+
+String, part of the upstream repository URL. The module provides sane defaults
+for all repositories. There should be no need to set this parameter.
+
+### `$ensure`
+
+Strings, allowed values are present, absent and latest. This sets the status of
```
Comment (alexjfisher) [inline README.md]: String instead of Strings?

File: README.md [hunk 2]
```diff
@@ -70,6 +71,46 @@ Fedora 25. Other nodesets are currently not present. The module requires at
...
+Boolean, if true, the module will enable the service. Defaults to true.
+
+### `$manage_repo`
+
+Boolean, if true, the module will setup the upstream repository. Defaults to
+true on Debian and CentOS family.
+
+### `$manage_jq`
+
+Boolean. jq is a dependency needed for the fact generation. This modules can
```
Comment (juniorsysadmin) [inline README.md]: module

File: README.md [hunk 3]
```diff
@@ -70,6 +71,46 @@ Fedora 25. Other nodesets are currently not present. The module requires at
...
+install it. Defaults to true on all systems except for CentOS. See also
+[limitations](#limitations).
+
+### `$repourl`
```
Comment (juniorsysadmin) [inline README.md]: If you wanted to be crystal clear you could have called it something like `$repourl_suffix`, but that's for another day.

Comment (bastelfreak) [reply README.md]: also it isn't a suffix, it is in the middle of the URL

File: README.md (hunk 2 above)
Comment (bastelfreak) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (bastelfreak) [reply README.md]: fixed

**Pull request merged**